default.json

```
@@ -164,12 +164,12 @@
     },
     {
       "matchManagers": ["npm"],
-      "matchDepNames": ["aws-sdk"],
-      "matchPackagePatterns": ["^@aws-sdk/", "^@smithy/"],
+      "matchPackageNames": ["aws-sdk"],
+      "matchPackagePatterns": ["^@aws-sdk/", "^@smithy/", "serverless"],
```

Assuming we only want to match on the `serverless` package itself, and not e.g. `serverless-plugin-xyz`:

```suggestion
      "matchDepNames": ["aws-sdk", "serverless"],
      "matchPackagePatterns": ["^@aws-sdk/", "^@smithy/"],
```